packages/shorebird_cli/lib/src/commands/release/releaser.dart

```diff
@@ -71,4 +73,22 @@ abstract class Releaser {
   Future<String> getReleaseVersion({
     required FileSystemEntity releaseArtifactRoot,
   });
+
+  /// Method for validating a public key argument, which will check, when
+  /// provided, that the file exists.
+  ///
+  /// This method should be called in [assertArgsAreValid] from releasers that
+  /// require a public key argument.
+  void assertPublicKeyArg() {
+    final patchSignKeyPath = argResults['public-key-path'] as String?;
+    if (patchSignKeyPath != null) {
+      final file = File(patchSignKeyPath);
+      if (!file.existsSync()) {
+        logger.err(
+          'No file found at $patchSignKeyPath',
+        );
+        exit(ExitCode.software.code);
+      }
+    }
```

Comment:
Could we potentially make this a more generic? ex: "we parsed argument `name` that should point to an existing file, but no file is at that path"? Then we could just call that directly from the releaser's `assertArgsAreValid` and potentially use it elsewhere.